cabal-install/Distribution/Client/Types.hs

```diff
@@ -151,6 +162,10 @@ instance Package ConfiguredId where
 instance Package (ConfiguredPackage loc) where
   packageId cpkg = packageId (confPkgSource cpkg)
 
+instance HasMungedId (ConfiguredPackage loc) where
```

Comment:
Should I everywhere rename `MungedId` to `MungedPackageId`? I don't think this is any more or less clear than the type itself.